### DISCUSSION TITLE
`staticDir` with addition headers

### DISCUSSION THREAD

**Discussion by derekdai** [HapticX/happyx#222] [General]
How can I add more headers for `staticDir`? I tried with `middleware` + `outHeaders` but no use.

**Comment by derekdai** [comment]
Thank you for the update, now it works.

One more question, will the `outHeaders` added in `middleware` affects `answerXXX`/`FileResponse` in non-`staticDir` routes (for consistency)?

**Reply by Ethosa** [reply]
> will the outHeaders added in middleware affects answerXXX/FileResponse in non-staticDir routes (for consistency)? 

nope
